src/Type/Object/PostObject.php

```diff
@@ -229,13 +290,15 @@ function get_post_object_fields( $post_type_object ) {
 					'description' => __( 'Format of the field output', 'wp-graphql' ),
 				],
 			],
-			'resolve' => function( $source, $args ) {
+			'resolve'     => function( $source, $args ) {
 				if ( isset( $args['format'] ) && 'raw' === $args['format'] ) {
+					// @codingStandardsIgnoreLine.
 					return $source->titleRaw;
 				} else {
+					// @codingStandardsIgnoreLine.
```

Comment:
this unnecessary else block an be omitted

Reply:
fixed